tests/puppeteer/pages/BO/orders/creditSlips/index.js

```diff
@@ -97,4 +103,18 @@ module.exports = class CreditSlips extends BOBasePage {
   async downloadCreditSlip(lineNumber = 1) {
     await this.page.click(this.creditSlipDownloadButton.replace('%ID', lineNumber));
   }
+
+  /**
+   * Generate PDF by date
+   * @param dateFrom
+   * @param dateTo
+   * @return {Promise<void>}
+   */
+  async generatePDFByDate(dateFrom = '', dateTo = '') {
+    if (dateFrom) {
```

Comment:
```suggestion
    if (dateFrom && dateTo) {
```
Why are you just checking for `dateFrom` here (maybe there's a reason ?) ?

Reply:
Because I think generally dateTO is the date of today, it's not mandatory to add it.
What do you think ?

Reply:
```suggestion
    if (dateFrom) await this.setValue(this.dateFromInput, dateFrom);
    if (dateTo) await this.setValue(this.dateToInput, dateTo);
```
Done ;)